### PR TITLE
MRG, FIX: Fix bug with rank checking in lcmv

### PR DESCRIPTION
**Pull request by larsoner**
@britta-wstnr feel free to try on your data

### CODE REVIEW

**Comment by britta-wstnr** [comment]
@larsoner works on my example with `rank=None` !

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -148,6 +148,8 @@ Bug
 
 - Fix the ``reduce_rank`` parameter in :func:`mne.beamformer.make_lcmv` and :func:`mne.beamformer.make_dics` to reduce the rank of the leadfield first before then reducing the rank of the denominator of the beamformer formula for the inversion by `Britta Westner`_ and `Eric Larson`_.
 
+- Fix bug with :func:`mne.beamformer.make_lcmv` with MaxFiltered data where rank was incorrectly checked by `Eric Larson`_.
+
```
Comment (britta-wstnr) [inline doc/changes/latest.inc]: I think it is not only for maxfiltered data. Could also happen with ICA'ed data etc - if no noise covariance matrix is supplied.

**Comment by agramfort** [comment]
ok to merge @britta-wstnr ? if so go for it.

**Comment by britta-wstnr** [comment]
thanks @larsoner :)

**Pull request merged**